tests/functional-tests/README.md

```diff
@@ -0,0 +1,77 @@
+# IBM Spectrum Scale CSI - Automation
```

Comment:
Suggest "Automation Testing"  as the heading ... or "Functional Test Automation" to match the directory name... 

IBM Spectrum Scale CSI should be implied being files under this repo